src/uu/sum/src/sum.rs

```diff
@@ -23,14 +23,19 @@ static USAGE: &str = "{} [OPTION]... [FILE]...";
 static SUMMARY: &str = "Checksum and count the blocks in a file.\n\
                         With no FILE, or when  FILE is -, read standard input.";
 
+// This can be replaced with usize::div_ceil once it is stabilized
+fn div_ceil(a: usize, b: usize) -> usize {
+    (a + b - 1) / b
```

Comment:
You should use 
```rust
        pub const fn div_ceil(a : usize, b: usize) -> usize {
            let d = a / b;
            let r= a % b;
            if r > 0 { // round up if not a multiple
                d + 1
            } else {
                d
            }
        }
```
to allow the compuler to use both the division and remainder. with only one asm instruction
(don't forget to use cargo fmt)

Reply:
When `b` is a power of two (as it is in this file), the original version versions better code. See https://godbolt.org/z/85fnajWhc

Reply:
@resistor yes, indeed the compiler looks smart enough in that case. My apologies.

Why not declairng the fonction constant though ?

Reply:
Done